shared/deps/keymap_to_qcode.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Please create a directory for these mapping files.

Reply:
updated, thanks~